### DISCUSSION TITLE
How to inspect key command or binding?

### DISCUSSION THREAD

**Discussion by Abdillah** [helix-editor/helix#2222] [Q&A]
Hello, I would like to know is there a way to dynamically print a command after pressing sequence of keys or a key. A case would be to help rebind it in the config file properly.

e.g. I try to rebind keys.selection hijk to other keys.

**Comment by CBenoit** [comment]
The closest we have currently is the command palette with `<space> ?`. Bind keys are shown between parenthesis. However, this doesn't show the command name, but the description.

You can search for the description in [`commands.rs`](https://github.com/helix-editor/helix/blob/master/helix-term/src/commands.rs#L193) and find the actual command name.

This isn't exactly ideal, and I think this is a good candidate for a feature request.

**Reply by pickfire** [reply]
Something like `xev` would be good and we can print out the keys users press, but not sure if it is useful to have something just for helix.
